MapboxMaps.podspec

```diff
@@ -12,19 +12,20 @@ Pod::Spec.new do |m|
   m.author = { 'Mapbox' => 'mobile@mapbox.com' }
   m.social_media_url = 'https://twitter.com/mapbox'
   m.documentation_url = 'https://docs.mapbox.com/ios/beta/maps/api-reference/'
-
-  m.source = { http: "https://api.mapbox.com/downloads/v2/mobile-maps-ios/releases/ios/packages/#{maps_version.to_s}/MapboxMaps.xcframework.zip" }
-  m.vendored_frameworks = 'MapboxMaps.xcframework'
-
+  
+  m.source = { :git => 'https://github.com/mapbox/mapbox-maps-ios.git', :tag => 'v10.0.0-beta.13' }
```

Comment:
Use the `maps_version` variable here to ensure consistency with the rest of the podspec. If I’m not mistaken, the release script would update the `maps_version` variable’s value but not this inlined value.

Reply:
correct, good call out